gubernator/pull_request_test.py

```diff
@@ -25,6 +25,10 @@ def make(number, version, result, start_time=1000):
     finished = result and {'result': result}
     return (number, started, finished)
 
+def makePodutil(number, version, result, start_time=1000):
```

Comment:
Nit: `version` -> `revision` for naming consistency

Reply:
otherwise lgtm

Reply:
done :+1: